src/test/java/api/requests/RequestScheduledNoticesProcessingTests.java

```diff
@@ -557,7 +558,10 @@ void titleLevelRequestExpirationNoticeShouldBeSentAndDeletedWithEnabledTlr() {
     final LocalDate localDate = getLocalDate().minusDays(1);
     final var requestExpiration = LocalDate.of(localDate.getYear(),
       localDate.getMonthValue(), localDate.getDayOfMonth());
-    IndividualResource request = requestsFixture.place(buildTitleLevelRequest(requestExpiration));
+    ItemResource smallAngryPlanet = itemsFixture.createMultipleItemsForTheSameInstance(1).get(0);
```

Comment:
try change Hold to Page in `buildTitleLevelRequest`, fix existing tests in this class

Reply:
done